hd/etc/perso_module/fratrie.txt

```diff
@@ -131,16 +131,21 @@
 %let;half_mother;%count;%in;
 %(aaa2 %half_father;, %half_mother;<br>%)
 
+%define;on_side(xx)
+  [*on %s's side:::xx]
+%end;
+
 %if;(op_m=1 and (half_father != 0 or half_mother != 0))
   <h2>[*half siblings]</h2>
   %if;(half_father != 0)
-        [*on %s's side:::] %apply;short_display_person("father")
+      %let;fff;%apply;very_short_display_person("father")%in;
+      %apply;on_side(fff)
```

Comment:
In that case, do you know what data breaks what? Maybe this should be fixed so that you can keep the `short_display_person` call?